### ISSUE TITLE
Old bower version?

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/calendar#855]
For https://github.com/nextcloud/activity/issues/277, I wanted to build latest master version for server master, but I get this:

```
tobi@tobi-laptop:/srv/http/nc/apps/calendar(master○) » make                                                                                                                                                                                                             127 ↵
make yarn
make[1]: Entering directory '/srv/http/nextcloud/apps/calendar'
cd js && /usr/bin/yarn run build
yarn run v1.7.0
$ yarn && node_modules/bower/bin/bower install && node_modules/bower/bin/bower update
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
bower hsl_rgb_converter#*   not-cached https://github.com/kayellpeee/hsl_rgb_converter.git#*
bower hsl_rgb_converter#*      resolve https://github.com/kayellpeee/hsl_rgb_converter.git#*
bower jstzdetect#*              cached https://github.com/georgehrke/jstimezonedetect.git#1.0.6
bower jstzdetect#*            validate 1.0.6 against https://github.com/georgehrke/jstimezonedetect.git#*
bower hsl_rgb_converter#*     download https://github.com/kayellpeee/hsl_rgb_converter/archive/master.tar.gz
bower hsl_rgb_converter#*      extract archive.tar.gz
bower hsl_rgb_converter#*     resolved https://github.com/kayellpeee/hsl_rgb_converter.git#3936a3896d
bower angular#1.6.8            EINVRES Request to https://bower.herokuapp.com/packages/angular failed with 502
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
make[1]: *** [Makefile:91: yarn] Error 1
make[1]: Leaving directory '/srv/http/nextcloud/apps/calendar'
make: *** [Makefile:86: build] Error 2
```

The website gives me:
This Bower version is deprecated. Please update it: npm install -g bower. The new registry address is https://registry.bower.io

But bower version is handled by this app, or?

**Comment by georgehrke** [comment]
Yes, it is: https://github.com/nextcloud/calendar/blob/master/js/package.json#L21
We really need to setup something like https://greenkeeper.io ... 🙈

**Comment by georgehrke** [comment]
fixed with #859
